src/ace/managers/blit.c

```diff
@@ -96,6 +98,16 @@ UBYTE _blitCheck(
 			);
 		}
 	}
+	#else
+	if(pSrc && pDst && bitmapIsInterleaved(pSrc) && bitmapIsInterleaved(pDst)) {
+		if(wHeight * pSrc->Depth > 32768) {
```

Comment:
that's ok but I dunno if I'd just do a single log msg regardless of OCS/non-OCS and just change the condition depending on define